src/js/components/atoms/icon/stories/icon.html

```diff
@@ -0,0 +1,2 @@
+<!-- @format -->
+<icon v-bind="$props" style="font-size: 32px;" />
```

Comment:
i don't like inline styles, can't we get rid of it?

Reply:
these are just for storybook

Reply:
👍